From 68cbcb7160c5b02c2184e850ff3d5c52864ba024 Mon Sep 17 00:00:00 2001
From: Gerrit Pape <pape@smarden.org>
Date: Tue, 25 Sep 2007 22:46:55 +0800
Subject: [PATCH] [MAN] Clarify description of -nt, -ot options to test builtin

Have the man page explicitely state how the test builtin behaves on
-nt and -ot options if file2 does not exist.  The case where file1
does not exist was already documented properly.

This was noticed by Sven Mueller and reported through
 http://bugs.debian.org/373611
---
 ChangeLog  |    4 ++++
 src/dash.1 |   12 ++++++++++--
 2 files changed, 14 insertions(+), 2 deletions(-)

diff --git a/ChangeLog b/ChangeLog
index d5c51bf..4ea7374 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,3 +1,7 @@
+2007-09-25  Gerrit Pape <pape@smarden.org>
+
+	* Clarify description of -nt, -ot options to test builtin.
+
 2007-09-25  Herbert Xu <herbert@gondor.apana.org.au>
 
 	* Do not expand tilde in parameter expansion within quotes.
diff --git a/src/dash.1 b/src/dash.1
index 00f40bd..3aefe21 100644
--- a/src/dash.1
+++ b/src/dash.1
@@ -1934,12 +1934,20 @@ exists and is a socket.
 .It Ar file1 Fl nt Ar file2
 True if
 .Ar file1
-exists and is newer than
+and
+.Ar file2
+exist and
+.Ar file1
+is newer than
 .Ar file2 .
 .It Ar file1 Fl ot Ar file2
 True if
 .Ar file1
-exists and is older than
+and
+.Ar file2
+exist and
+.Ar file1
+is older than
 .Ar file2 .
 .It Ar file1 Fl ef Ar file2
 True if
-- 
1.5.5

